doc/tutorials/enable_s5.rst

```diff
@@ -290,6 +290,7 @@ How to Test
       sudo python3 ~/user_vm_shutdown.py <User VM name>
 
    .. note:: The User VM name is configured in the :file:`life_mngr.conf` of User VM.
+   .. note:: For the WaaG VM, the User VM name is "windows".
```

Comment:
Better would be to combine these two notes into one:

```suggestion
      For the WaaG VM, the User VM name is "windows".
```

Reply:
Thanks, will update this. BTW, I think current config tool doesn't support configure communication vuart for s5 in this document. @NanlinXie Can you help double check this? thanks!